apps/meteor/client/polyfills/hoverTouchClick.ts

```diff
@@ -1,17 +0,0 @@
-import * as domEvents from '../lib/utils/domEvents';
-import { isIOsDevice } from '../lib/utils/isIOsDevice';
```

Comment:
The helper `isIosDevice` is not being used anywhere else in the project, we should remove that as well.

Reply:
Even better! More deleted lines!

Reply:
Done!